packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -23,17 +21,25 @@ let leaf;
 let claimerProof;
 let windowIndex;
 
+const sampleIpfsHash = "QmfVMHgoWpTSZqovo7vhM7Wcmz6EeX4QBYbCk4DZTNM8u3";
+
 // For a recipient object, create the leaf to be part of the merkle tree. The leaf is simply a hash of the packed
 // account and the amount.
 const createLeaf = recipient => {
   assert.isTrue(
     Object.keys(recipient).every(val => ["account", "amount", "accountIndex"].includes(val)),
     "recipient does not contain required keys"
   );
-  return web3.utils.soliditySha3(
-    { t: "address", v: recipient.account },
-    { t: "uint256", v: recipient.amount },
-    { t: "uint256", v: recipient.accountIndex }
+
+  return Buffer.from(
```

Comment:
Why'd you add Buffer.from wrapper?

Reply:
the typescript library returns a buffer. the previous one returned a string.